view/frontend/web/css/jpay.css

```diff
@@ -32,3 +31,78 @@
   float: right;
   margin: 15px 30px;
 }
+
+.payment-method.jpay .jpay-logo-eg {
+    height: 61px;
+    width: 200px;
+    background: no-repeat url('../images/secured_payfac_eg.png');
+    background-position-y: 2px;
+    background-size: 100% 100%;
+    border: 1px solid #DDD;
+    border-radius: 3px;
+    border-color: transparent;
+    float: right;
+    margin-right: 5px;
+}
+
+.payment-method.jpay .jpay-logo-ke {
+    height: 61px;
+    width: 200px;
+    background: no-repeat url('../images/secured_payfac_kenya.png');
```

Comment:
Can we have a common pattern on country name ? ex 
`../images/secured_payfac_ke.png`